### PR TITLE
feat: add review section to service page

### PR DESCRIPTION
**Pull request by jthistle**
resolves: https://trello.com/c/oVae6bEX/

Looks like:

Screenshot from 2021-03-16 15-11-42

### CODE REVIEW

**Comment by marxian** [comment]
#### 📝Changed routes:


| [**/service/\[id\]**](https://my-best-life-9kj2cid5f-neontribe.vercel.app/service/[id]) |
|:-:|
|<a href="https://my-best-life-9kj2cid5f-neontribe.vercel.app/service/[id]">Screenshot of /service/[id]</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/i9gHAF70PQEAlvdJUgTEMY6R_qI_.png">(view full size)</a> |







Commit 36949063daa95c3e3c1db8101964b421b5bda4fb (https://my-best-life-9kj2cid5f-neontribe.vercel.app).

**Review by edpe — APPROVED** [review]
Works well :star: :star: :star: :star: :star:

File: pages/service/[id].tsx [hunk 1]
```diff
@@ -330,6 +367,29 @@ export const ServicePage = ({ serviceData }: ServicePageProps): JSX.Element => {
...
+        <FiveStar
+          currentRating={reviewState.rating}
+          onChange={onRatingChange}
+        />
+        <Checkbox
+          label={'I have attended this service'}
+          checked={reviewState.hasUsedService}
+          onChange={onUsedServiceChange}
+        />
+        <span>Leave a review</span>
```
Comment (Nikomus) [inline pages/service/[id].tsx:381]: Can we change this one to a label that wraps the input?

File: pages/service/[id].tsx [hunk 2]
```diff
@@ -330,6 +367,29 @@ export const ServicePage = ({ serviceData }: ServicePageProps): JSX.Element => {
...
+        <TextInput
```
Comment (Nikomus) [inline pages/service/[id].tsx:382]: textarea can take a `rows` attribute that guarantees the height matches the text inside exactly

**Pull request merged**